packages/html/ds/tsconfig.json

```diff
@@ -19,7 +19,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true,
-    "types": ["vitest/globals"]
+    "types": ["vitest/globals", "@testing-library/jest-dom"]
```

Comment:
Added this to not raise typescript errors in test files when building the html/ds project.